### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by subutai**
for V1.0

### CODE REVIEW

**Comment by subutai** [comment]
@BoltzmannBrain FYI. Don't merge yet though.

File: README.md [hunk 1]
```diff
@@ -89,67 +79,80 @@ This will install the additional required modules pandas and simplejson.
...
 
-This repo contains a corpus of 32 data files of time-series data. The format of the CSV files is specified in Appendix F of the [NAB Whitepaper](https://github.com/numenta/NAB/wiki#nab-whitepaper). The detector under test will read in, and be scored on, all data files in the corpus. The format of results files is also specified in the whitepaper posted in the wiki.
+This will run the Numenta detector only and produce normalized scores. Note that
+by default it tries to use all the cores on your machine. This should take about
+20-30 minutes on a current powerful laptop with 4-8 cores.  For debugging
+you can run subsets of the data files by modifying and specifying specific label
+files. Please type:
 
-##### Data and results visualization
+   python run.py --help
```
Comment (BoltzmannBrain) [inline README.md]: indent

File: README.md [hunk 2]
```diff
@@ -89,67 +79,80 @@ This will install the additional required modules pandas and simplejson.
...
 
-##### Run NAB
+There are several different use cases for NAB:
 
-    cd /path/to/nab
-    python run.py
+1. If you just want to look at all
+the results we reported in the paper, there is no need to run anything.
+All the data files are in the data subdirectory and all individual detections
+for reported algorithms are checked in the results subdirectory. Please see
```
Comment (BoltzmannBrain) [inline README.md]: "are checked in to the results..."

File: README.md [hunk 3]
```diff
@@ -77,8 +68,7 @@ You need to manually install the following:
 
 ##### Download this repository
 
-    cd ~/
-    git clone https://github.com/numenta/NAB.git
+    Use the Github links provided in the right sidebar.
```
Comment (BoltzmannBrain) [inline README.md]: This seems odd indented like code.

**Comment by BoltzmannBrain** [comment]
@subutai done w/ review

File: README.md [hunk 4]
```diff
@@ -12,8 +12,9 @@ Included are the tools to allow you to easily run NAB on your
 own anomaly detection algorithms; see the [NAB entry points
 info](https://github.com/numenta/NAB/wiki#nab-entry-points). Competitive results
 tied to open source code will be posted in the wiki on the
-[Scoreboard](https://github.com/numenta/NAB/wiki#nab-scoreboard). Let us know
-about your work by submitting a pull request.
+[Scoreboard](https://github.com/numenta/NAB/wiki/NAB%20Scoreboard). Let us know
```
Comment (breznak) [inline README.md]: a nit, I'd go with name w/o %20

File: scripts/README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+in hand labeling datasets. To use it do the following:
+
+First generate the list of data files and result files:
+
+    cd /path/to/nab
+    ls -1 data/*/*.csv | grep data > scripts/data_file_paths.txt
+    ls -1 results/*/*/*.csv | grep results | grep -v test_results > scripts/results_file_paths.txt
+    cd scripts
+    ln -s ../data
+    ln -s ../results
```
Comment (breznak) [inline scripts/README.md]: is this correct? you're missing 2nd argument (I don't know what it does w/ just 1)

File: scripts/README.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+From the scripts directory, type:
+
+    python -m SimpleHTTPServer 12345
+ 
+Then, open Chrome (only works on Chrome!)and type this into the url window:
```
Comment (BoltzmannBrain) [inline scripts/README.md]: space missing

File: scripts/README.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+ 
+    localhost:12345/nab_visualizer.html
+ 
+To view data, click on "look at data", click in query window and then
+press RETURN key. This should show all the data files. You can also filter
+the files by keyword with the query window; it will filter for filenames that
+contain the entered characters.
```
Comment (BoltzmannBrain) [inline scripts/README.md]: add "(case-sensitive)"

File: README.md [hunk 8]
```diff
@@ -29,7 +30,7 @@ on Machine Learning and Applications, December 2015.
```
Comment (BoltzmannBrain) [inline README.md]: In the citation above please change my name to Alexander :smile:

File: README.md (hunk 4 above)
Comment (subutai) [inline README.md]: That's the link to the scoreboard. In urls %20 is just a space.

File: scripts/README.md (hunk 5 above)
Comment (subutai) [inline scripts/README.md]: Yes, that's correct. Without second argument it just uses the original name.

**Pull request merged**